app/controllers/api/v1/screenshots_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    # Controller methods to call a service of AWS S3
+    class ScreenshotsController < ApplicationController
+      before_action :get_versions_id, :get_locales_id
+
+      def index
+        @screenshots = Screenshot.where(version_id: @version, locale_id: @locale)
+        render json: @screenshots, include: :images
+      end
+
+      def new
+        @screenshot = Screenshot.new
+      end
+
+      def create
+        @screenshot = Screenshot.new(screenshot_params)
+        @screenshot.save
```

Comment:
Error handling required here. `save` will not report errors even if it's technically invalid. You can use something like `save!`. This will atleast give you an error. 

To handle the exceptions read:
https://api.rubyonrails.org/classes/ActiveModel/Errors.html
https://guides.rubyonrails.org/active_record_validations.html